src/rule/index.ts

```diff
@@ -195,6 +240,10 @@ const verifyRequest = (
     });
   }
 
+  if (request.postData) {
+    matchRequestBody(operationNode, request, resultOperationNode, result);
+  }
```

Comment:
This does not cover the case that the spec says the request should have a body but it does not?

Reply:
Added a test and check for this!